api.yaml

```diff
@@ -226,6 +246,21 @@ definitions:
     required:
       - id
       - content
+  Device:
+    title: Device
+    description: Device data.
+    type: object
+    properties:
+      installationId:
+        type: string
```

Comment:
is this supposed to be stable and unique per app-installation? (regardless of the platform) 

if that's the case, perhaps a better resource scheme could be:

* `PUT /v1/device/INSTALLATION_ID` (registration)
* `DELETE /v1/device/INSTALLATION_ID` (de-registration)